server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java

```diff
@@ -21,7 +21,7 @@
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 
-import reactor.core.publisher.Mono;
+import org.reactivestreams.Publisher;
```

Comment:
not a java.util.concurrent.Flow.Publisher?

Reply:
Give me Java 9 then (: 
reactivestreams is the api jar that allows to have `Publisher` before Java 9.